mm2src/common/common.rs

```diff
@@ -661,6 +661,16 @@ pub fn now_float() -> f64 {
     duration_to_float(Duration::from_millis(now_ms()))
 }
 
+pub fn wait_until_sec(seconds: u64) -> u64 { (now_ms() / 1000) + seconds }
+
+pub fn wait_until_ms(milliseconds: u64) -> u64 { now_ms() + milliseconds }
+
+pub fn now_sec() -> u64 { now_ms() / 1000 }
+
+pub fn now_sec_u32() -> u32 { (now_ms() / 1000) as u32 }
+
+pub fn now_sec_i64() -> i64 { (now_ms() / 1000) as i64 }
```

Comment:
This is used primarily for sqlite db operations where [integer is represented as i64](https://docs.rs/sqlite/latest/sqlite/enum.Value.html#variant.Integer)

> but I suspect it can create edge cases if we are doing i64<->u32 in many different places.

Even if it's converted to u32, we still have until 2106 to fix this :)

We should at the very least use `TryFrom` with `expect` to panic with a message. Will fix it, I will use `expect` since returning errors everywhere will be excessive at this stage.

Reply:
Partially fixed in https://github.com/KomodoPlatform/atomicDEX-API/pull/1827